style.css

```diff
@@ -0,0 +1,36 @@
+div #main{
+    float:left;
+}
+#main div{
+    float: left;
+    width:50%;
+}
+#main div #navbar{
+width:40% ;
+} 
+#nav-list li {
+    text-align: center;
+    list-style: none;
+    border-style: solid;
+    margin: 1px;
+    border-top-color:transparent;
+    
+    
+}
```

Comment:
Kindly include a text decoration of none, to remove the decorations and make the project presentable. 
Also make sure to remove the blank spaces for the code to be clean.

Reply:
Hi @Rn486 ✋

- [ ] [Optional] Your review is polite and well explained. Try to use emojis as they make your comments more human-friendly